components/Map.vue

```diff
@@ -96,11 +124,9 @@ export default {
           )
         })
 
-        if (businesses.length) {
```

Comment:
How come this check no longer needed?

Reply:
This might be resolved by merging, I think.  The check is there now.